### PR TITLE
Test `Cli Model` functionalities

### PR DESCRIPTION
**Pull request by safoinme**
## Describe changes
Initial Tests for cli model. 

## Pre-requisites
Please ensure you have done the following:
- [ ] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/component-gallery/integrations) table and the [corresponding website section](https://zenml.io/integrations).
- [ ] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

File: tests/integration/functional/cli/test_model.py [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+):
+    # Patch the find_model_server method
+    mocker.patch(
+        "zenml.integrations.mlflow.model_deployers.mlflow_model_deployer.MLFlowModelDeployer.find_model_server",
+        return_value=[
+            mlflow_service_mock[0]
+        ],  # The return value of the patched method
+    )
+
+    # Get the list command
```
Comment (bcdurak) [inline tests/integration/functional/cli/test_model.py:124]:
```suggestion
    # Get the describe command
```

File: tests/integration/functional/cli/test_model.py [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
```
Comment (bcdurak) [inline tests/integration/functional/cli/test_model.py:153]:
```suggestion
    # Get the get-url command
```

File: tests/integration/functional/cli/test_model.py [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+def test_describe_model(
+    mocker,
+    mlflow_service_mock,
+    mlflow_model_deployer,
+):
+    # Patch the find_model_server method
+    mocker.patch(
+        "zenml.integrations.mlflow.model_deployers.mlflow_model_deployer.MLFlowModelDeployer.find_model_server",
+        return_value=[
+            mlflow_service_mock[0]
```
Comment (bcdurak) [inline tests/integration/functional/cli/test_model.py:120]: Why do we have to create another list just containing the first element here? Can we just use the `mlflow_service_mock` as the return value?

File: tests/integration/functional/cli/test_model.py [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+        service.endpoint.url = f"http://test_url_{i}"
+        services.append(service)
+    return services
+
+
+@pytest.fixture
+def mlflow_model_deployer(
+    clean_project,
+):
+    """Returns a local orchestrator."""
```
Comment (bcdurak) [inline tests/integration/functional/cli/test_model.py:75]:
```suggestion
    """Returns an mlflow model deployer."""
```

**Review by bcdurak — COMMENTED** [review]
The tests look good overall. I have left a few small comments.

File: tests/unit/cli/test_model.py [hunk 5]
```diff
@@ -0,0 +1,414 @@
...
+
+    # Run the command
+    runner = CliRunner()
+    result = runner.invoke(
+        get_logs,
+        [str(concrete_service_mock[0].uuid)],
+        obj=concrete_model_deployer,
+    )
+    assert result.exit_code == 0
+    assert "" in result.output
```
Comment (strickvl) [inline tests/unit/cli/test_model.py:414]: What's the purpose of this?

**Pull request closed without merging**